NEWS.md

```diff
@@ -1,4 +1,22 @@
-## 5.6.0 - Unreleased
+## 5.7.0 - Unreleased
+
+## 5.6.0 - Orchid R1 2023
+The focus of this release was to migrate to Vertx future and improve logging 
+
+[Full Changelog](https://github.com/folio-org/mod-invoice/compare/v5.5.0...v5.6.0)
+
+### Stories
+* [FOLIO-1021](https://issues.folio.org/browse/FOLIO-1021) - Update each README to use range of years in the copyright statement
+* [FOLIO-3678](https://issues.folio.org/browse/FOLIO-3678) - Ensure API-related GitHub Workflows are used for relevant backend repositories
```

Comment:
I think we should not include first two in the news fiile

Reply:
Done